tests/utilities/test_re.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from phoenix.utilities.re import parse_env_headers
+
+
+@pytest.mark.parametrize(
+    "case",
+    [
+        # invalid header name
+        ("=value", [], True),
```

Comment:
```suggestion
        pytest.param("=value", [], True, id="invalid-header-name-1"),
```

Suggestion: Using `pytest.param` with an ID makes it easier to run individual parameterized test cases and view test results in both IDEs and when running `pytest` from the command line.

Reply:
Good to know - unfortunately I don't really know why these are bad cases just right now since I haven't read the spec. I'm gonna leave for now.

Reply:
oh I see the name is in the comment nvm